pyrobosim_ros_gym/envs/__init__.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
+
+from pyrobosim_ros_gym.envs.banana import BananaEnv
+from pyrobosim_ros_gym.envs.greenhouse import GreenhouseEnv
+from pyrobosim_ros_gym.envs.pyrobosim_ros_env import PyRoboSimRosEnv
+
+CLASS_BY_NAME = {c.__name__.replace("Env", ""): c for c in [BananaEnv, GreenhouseEnv]}
```

Comment:
I wouldn't be so overly clever, and just list them out manually. Would also change the variable name. Something like:

```python
ENV_CLASS_FROM_NAME = {
    "Banana": BananaEnv,
    "Greenhouse": GreenhouseEnv,
}